os/arch/arm/src/armv7-m/up_busfault.c

```diff
@@ -0,0 +1,148 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * arch/arm/src/armv7-m/up_busfault.c
+ *
+ *   Copyright (C) 2011, 2013 Gregory Nutt. All rights reserved.
+ *   Author: Gregory Nutt <gnutt@nuttx.org>
```

Comment:
Is there any code you refer or get from nuttx? In other words, do you need to add Nuttx license?

Reply:
We made a copy of up_memfault and modified it to handle bus fault. But, I think its quite different from nuttx code. It is better to remove the nuttx copyright. @abhishek-samsung  Pls remove it.

Reply:
Done.